packages/frontend/src/components/MenuBar/MenuItem.tsx

```diff
@@ -124,7 +124,7 @@ const MenuItemContent = ({
             <ExternalLinkIcon className={styles.arrowIcon} />
           ) : count ? (
             <div className={styles.badgeWrapper}>
-              <Badge label={count} variant={isInbox ? 'strong' : 'neutral'} />
+              <Badge label={count} />
```

Comment:
Badge has `color` prop, either `subtle` (this is the same as `neutral`) or `alert` (this is the same as `strong`):
https://altinn.github.io/altinn-components/?path=/story/atoms-badge-badge--default&args=color:subtle